iina/PrefKeyBindingViewController.swift

```diff
@@ -71,47 +95,12 @@ class PrefKeyBindingViewController: NSViewController, PreferenceWindowEmbeddable
       useMediaKeysButton.title = NSLocalizedString("preference.system_media_control", comment: "Use system media control")
     }
 
-    // config files
-    // - default
-    PrefKeyBindingViewController.defaultConfigs.forEach {
-      userConfigNames.append($0.key)
-    }
-    // - user
-    guard let uc = Preference.dictionary(for: .inputConfigs)
-    else  {
-      Logger.fatal("Cannot get config file list!")
-    }
-    userConfigs = uc
-    userConfigs.forEach {
-      userConfigNames.append($0.key)
-    }
-
-    var currentConf = ""
-    var gotCurrentConf = false
-    if let confFromUd = Preference.string(for: .currentInputConfigName) {
-      if getFilePath(forConfig: confFromUd, showAlert: false) != nil {
-        currentConf = confFromUd
-        gotCurrentConf = true
-      }
-    }
-    if !gotCurrentConf {
-      currentConf = "IINA Default"
-    }
-    // load
-    confTableSelectRow(withTitle: currentConf)
-    currentConfName = currentConf
-    guard let path = getFilePath(forConfig: currentConf) else { return }
-    currentConfFilePath = path
+    // Load the config file saved in user default
+    loadConfigFile(Preference.string(for: .currentInputConfigName))
```

Comment:
Previously there was no `loadConfigFile()` here. So now it calls `setKeybindingsForPlayerCore()` when the view is loaded. Not a big issue, but we need to be careful when the behavior changes.

Reply:
Added a guard to prevent loading the same config